examples/ampcontext-creative.html

```diff
@@ -2,7 +2,96 @@
 <html>
   <head>
     <style>body { background: lightblue; }</style>
-    <script src="./ampcontext-creative.js"></script>
+    <script>
+      if (!window.context){
+        // window.context doesn't exist yet, must perform steps to create it
+        // before using it
+        console.log("window.context NOT READY");
+
+        // must add listener for the creation of window.context
+        window.addEventListener('amp-windowContextCreated', function(){
+          console.log("window.context created and ready to use");
+          window.context.onResizeSuccess(resizeSuccessCallback);
+          window.context.onResizeDenied(resizeDeniedCallback);
+        });
+
+        // load ampcontext-lib.js which will create window.context
+        ampContextScript = document.createElement('script');
+        ampContextScript.src = "../dist.3p/current/ampcontext-lib.js";
+        document.head.appendChild(ampContextScript);
+      }
+
+      function intersectionCallback(payload){
+        changes = payload.changes;
+        // Step 4: Do something with the intersection updates!
```

Comment:
Nit: Still need to do something with "Step 4", b/c there's no 1..3 in this file.